ProxyAppsSuite/tests/miniamr.yaml

```diff
@@ -0,0 +1,896 @@
+_base:
+   summary: 'Apply a stencil calculation on a unit cube computational domain'
+   build:
+      summary: building the miniAMR test
+      modules: [ gcc, openmpi ]
+      env:
+         CC: mpicc
+      cmds:
+          - git clone https://github.com/Mantevo/miniAMR
+          - cd miniAMR/ref 
+          - make || exit 1
+   scheduler: slurm
+   subtitle: "{{test_input.id}}"
```

Comment:
since each version only kicks off one test each, I don't think a subtitle is necessary here (unless you anticipate that we'll find new values to test and eventually permute over `test_input` in the future)